README.md

```diff
@@ -28,7 +28,7 @@ couple of `fancy` features.
 - __daemon.py__ : Daemon class written by Sander Marechal (http://www.jejik.com) to start a python script as a daemon.
 - __missing.py__: Small functions and tools that are commonly used but not
   available in the Python (2.x) API.
-- __cache.py__ : File cache to store pickled data identified by a key accompanied by a timestamp,
+- ~~__cache.py__ : File cache to store pickled data identified by a key accompanied by a timestamp.~~ Was moved to [vsc-utils](https://github.com/hpcugent/vsc-utils)
```

Comment:
Grammar-wise: no subject in the `Was` sentence, if you need a reason to change it.